src/Framework/LazyFormattedBuildEventArgs.cs

```diff
@@ -91,7 +90,6 @@ public LazyFormattedBuildEventArgs
         protected LazyFormattedBuildEventArgs()
             : base()
         {
-            locker = new Object();
```

Comment:
This is music to my ears, we save on allocations as well as 4 (or 8) bytes per args instance.